dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-adoption-insights/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "red-hat-developer-hub-backstage-plugin-adoption-insights",
+  "version": "0.2.0",
```

Comment:
A new version is now available `0.2.1` in [npm](https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-adoption-insights)